### ISSUE TITLE
[A11y] cannot control dropdown component with keyboard in Firefox

### ISSUE THREAD

**Issue by shyfruan** [carbon-design-system/carbon-components-react#406]
## Detailed description

The item inside component `dropdown` cannot be entered for Firefox in mac;
The opened list cannot be disappeared after one item selected for Firefox in windows.

The issue can be reproduced in http://react.carbondesignsystem.com/?selectedKind=Dropdown&selectedStory=with%20default%20text&full=0&down=1&left=1&panelRight=0&downPanel=storybook%2Factions%2Factions-panel

Dropdown component works fine in Chrome. 

> Is this issue related to a specific component?

Dropdown

> What browser are you working in?

Firefox

> What version of the Carbon Design System are you using?

3.10.6

## Add labels
a11y

**Comment by marijohannessen** [comment]
This will be resolved with the release of the ListBox component, right @joshblack ?

**Comment by joshblack** [comment]
Yupp! @marijohannessen

**Comment by flyHe** [comment]
it seems this issue has not been fixed in the carbon-componets-react 5.4.0 release

**Comment by joshblack** [comment]
@flyHe correct, we haven't merged in the work for Dropdown v2 yet 😄 

Will close this issue with the corresponding PR when it's up!

**Comment by joshblack** [comment]
Closed by #480

**Comment by flyHe** [comment]
still can not control dropdown component with keyboard in Firefox
http://react.carbondesignsystem.com/?selectedKind=Dropdown&selectedStory=with%20default%20text&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

**Comment by asudoh** [comment]
@flyHe I guess @joshblack meant DropdownV2: http://react.carbondesignsystem.com/?selectedKind=DropdownV2&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

**Comment by flyHe** [comment]
@asudoh @joshblack Can you control dropdown or dropdownV2 component with keyboard in Firefox? 
System: MacOS v10.13.2
In chrome Version 63.0.3239.132 (Official Build) (64-bit), with the link [dropdown](http://react.carbondesignsystem.com/?selectedKind=Dropdown&selectedStory=with%20default%20text&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel) and [DropdownV2](http://react.carbondesignsystem.com/?selectedKind=DropdownV2&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel),
I can control dropdown component with keyboard and can not control dropdownV2 component with keyboard
In firefox Version ESR 52.6.0 (64-bit),with the link [dropdown](http://react.carbondesignsystem.com/?selectedKind=Dropdown&selectedStory=with%20default%20text&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel) and [DropdownV2](http://react.carbondesignsystem.com/?selectedKind=DropdownV2&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel),
I can not control both dropdown and dropdownV2 component with keyboard

**Comment by tw15egan** [comment]
I am able to control both the Dropdown and DropdownV2 in Firefox ESR, 52.6
